core/models/quota.py

```diff
@@ -146,12 +146,6 @@ def default_dict(cls):
     class Meta:
         db_table = 'quota'
         app_label = 'core'
-        # unique_together = ("cpu", "memory", "storage",
-        #                    "floating_ip_count",
-        #                    "port_count",
-        #                    "instance_count",
-        #                    "snapshot_count",
-        #                    "storage_count")
```

Comment:
NO MORE! (this isn't the commit that disable them btw, i'm just deleting the comments).

I removed them for allocation as well. There's no reason to force them to be unique. It adds the overhead of having to check if they exist before assigning them to a user. This is going to make for intense queries later on.